### PR TITLE
gallery pagination

### PR DESCRIPTION
**Pull request by wunaidage**
also fixed malayalam_class link on index page

### CODE REVIEW

File: gallery.html [hunk 1]
```diff
@@ -256,6 +298,49 @@
...
+        var pageCount = Math.ceil(items.length/5);
+        var first = template('|<<', -1);
+        var last = template('>>|',0);
+
+        $(items).hide();
+        for(var i = 0; i < 5; i++){
+            $(items[i]).show();
+        }
+
+        $('#pagination').find('ul').append(first);
```
Comment (saba83rish) [inline gallery.html:313]: oh we have to build the pagination logic..i thought there might been some ready made code for it already

**Review by saba83rish — APPROVED** [review]

File: gallery.html (hunk 1 above)
Comment (wunaidage) [reply gallery.html:313]: yeah, i just don't want to study... and the libs has their own styles. don't want to bother to change the style things

Comment (wunaidage) [reply gallery.html:313]: need 20 mins to write own but need 40 mins to learn how to use libs...

Comment (saba83rish) [reply gallery.html:313]: haha ok..makes sense..mergin it

**Pull request merged**